dockers/docker-snmp/snmpd.conf.j2

```diff
@@ -51,8 +72,16 @@ rocommunity6 {{ snmp_rocommunity }}
 
 #  Note that setting these values here, results in the corresponding MIB objects being 'read-only'
 #  See snmpd.conf(5) for more details
-sysLocation    {{ snmp_location }}
+{% if SNMP is defined and SNMP.LOCATION is defined %}
+sysLocation    {{ SNMP.LOCATION.Location }}
+{% else %}
+sysLocation    public
```

Comment:
Is it possible to remove the default, like community?

Reply:
I can leave this blank. Suvarna asked me in the PR earlier to have a default. I'm fine with however you want it.

Reply:
Per our conversation this was ok to leave as is.